pkg/controller/jobs/generic/generic_multikueue_adapter_test.go

```
@@ -0,0 +1,276 @@
+			featureEnabled: true,
+			want:           false,
+			wantReason:     "Expecting .spec.managedBy to be \"kueue.x-k8s.io/multikueue\" not \"not-a-string\"",
+			wantErr:        false,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Save original feature gate state
+			originalValue := features.Enabled(features.MultiKueueAdaptersForCustomJobs)
+			features.SetEnable(features.MultiKueueAdaptersForCustomJobs, tt.featureEnabled)
```

golangci-lint (2.2.2)

Error return value of `features.SetEnable` is not checked

(errcheck)